config/default.json

```diff
@@ -1,43 +1,26 @@
 {
     "routes": [
         {
-            "name": "name1",
             "proxies": [
                 {
-                    "name": "proxy1",
-                    "binding": "search",
-                    "url": "http://www.google.com",
-                    "folder": "google"
+                    "binding": "date",
+                    "url": "http://date.jsontest.com",
```

Comment:
Here is where you add the variables from lookup that you want to use. Just add {{variable}} to the url. The **query**, **search** and **path** are pre-defined, the other variables are taken from ```paths``` array that matches the current request. These variables are added after these are defined, so using the same name will overwrite them